workflows/README.md

```diff
@@ -0,0 +1,310 @@
+
+> "But the command is very short, e.g. `eslint . --ext ts --ext tsx`".
+>
+> When this command needs to be updated, it now needs to update in all the places in the documentation, the workflow files, ánd in the _minds of the maintainers_. Extracting this to a script resolves all that. Reading a workflow file can also be **very** daunting.
+
+### PR exercises changed
+
+The `scripts/pr` and `scripts/pr-check` scripts are ran with arguments: each file that has been changed or added in this PR. For example, if `two-fer` has been updated, a call might look like this:
+
+```bash
+scripts/pr exercises/two-fer/README.md exercises/two-fer/.meta/example.ext
+```
+
+It's recommended to run any actions against the changed _exercise_ and not the changed _file_. This is because changing a file is likely to trigger changes for the entire exercise (think: configuration, packages).
+
+> **Not ready?** / **Complex?**
+>
+> Before implementing this optimization, it may be ignored! The migration guide hints at adding at a later stage. If the input arguments are ignored, all the checks are running on all the exercises. **This is perfectly fine**. It will just take longer.
+
+### Integrity
+
+If the track has a single "top-level" dependency file and/or other configuration files, add an [integrity][wiki-integrity] step (to exist alongside a `scripts/sync` or `bin/sync`, which would copy all configuration files to all exercises) that ensures the top-level/base files are the same as the one copied to the exercise directories. Now dependencies can be updates, synced across the repository, and ensures that all exercises have the same configuration.
+
+A common way to accomplish this is to use a checksum. Ubuntu (and various other Linux distributions) comes with a tool called `sha1sum`, but using _whichever_ method to hash or reduce the configuration file (md5, sha1, crc32) to a checksum value, would work:
+
+```bash
+$ sha1sum README.md
+cd58091c5043bf21f00d39ff1740d8b2976deeff *README.md
+```
+
+### Security
+
+If the track uses additional workflows that require access to the GitHub token or other secrets, it's best practice to pin **all** actions used in the workflow to a specific commit. See [GitHub's security hardening guide][github-actions-security] for details.
+
+For example:
+
+```diff
+- uses: julia-actions/setup-julia@v1
++ uses: julia-actions/setup-julia@d26d1111976eae5f00db04f0515ab744ec9cd79e # 1.3.1
+```
+
+If the tooling has lockfiles for dependency management, consider checking it into the repository and use a "frozen lockfile" inside the workflow files. For example: `npm ci`, `yarn install --frozen-lockfile` and `bundle install --frozen`. This ensures that the lockfile is up-to-date when changing dependencies and prevents malicious packages to come in.
```

Comment:
I'm not sure we care, as the point is clear, but Bundlers `--frozen` flag is deprecated.

Reply:
What's the "new" way?

Reply:
`bundler config set x y` (e.g. https://github.com/exercism/v3-website/blob/master/docker/rails.Dockerfile#L16)

Basically, set it globally, rather than just on the one command.